tendrl/commons/flows/expand_cluster/utils.py

```diff
@@ -0,0 +1,58 @@
+import copy
```

Comment:
ahh, I missed that, you can add "include_current_node" as a flag for the original function and re-use it as you had suggested